rail_scripts/install-pz-rail

```diff
@@ -31,6 +31,15 @@ CFLAGS="-L $CONDA_PREFIX/lib -I $CONDA_PREFIX/include" pip install fitsio
 CFLAGS="-O2 -DNDEBUG -std=gnu99" pip install pytdigest
 pip install flexcode
 
+# copied from the old requirements.txt 
```

Comment:
there is a open [PR](https://github.com/linea-it/pz-compute/pull/71/files) adding two more requirements: joblib, pz_rail in the old requirement.txt